_posts/2022-10-03-project-maturity-levels.md

```diff
@@ -0,0 +1,70 @@
+---
+
+date: 2022-10-03 00:00:00-0700
+categories: blog
+author: Björn Kimminich
+author_image: /assets/images/people/leader_bjoern.jpg
+layout: blogpost
+title: Introducing new "Production" project maturity level
+excerpt_separator: <!--more-->
+
+---
+
+In order to distinguish [projects](https://owasp.org/projects/) more clearly over their lifecycle, OWASP has introduced a new _Production_ maturity level. It offers a natural and final step for all projects of sufficient maturity and activity after _Lab_ status, and allows to treat _Flagship_ finally as the strategic bonus level it was always meant to be. Along with the new level, a clear guidance on progression requirements and the promotion process have been documented by [the Project Committee](https://owasp.org/www-committee-project/).    
+
+<!--more-->
+
+## OWASP Project Maturity Levels
+
+All new OWASP projects start at the _Incubator_ level, on which they are considered experimental, proof-of-concept, work-in-progress or sometimes just an idea. Once proven themselves with a stable release, introduction of good open source development & maintenance practices and being received positively by the community, projects would request promotion to _Lab_ level. Up until now, this was the end of the line for many projects, as the _Flagship_ level has been scarecely awared to projects of strategic value to the OWASP mission. Ending up on a level called _Lab_ felt somewhat unnatural for very mature and active projects. Entering: _Production_ level.
```

Comment:
"work-in-progress, or sometimes" - add an oxford comma (@kingthorin will hopefully be proud of me;)